android/src/main/java/com/stripeterminalreactnative/Mappers.kt

```diff
@@ -487,3 +491,17 @@ fun mapFromReceiptDetails(receiptDetails: ReceiptDetails?): ReadableMap =
         putString("tsi", receiptDetails?.tsi)
         putString("tvr", receiptDetails?.tvr)
     }
+
+internal fun mapFromNetworkStatus(status: NetworkStatus): String {
+    return when (status) {
+        NetworkStatus.ONLINE -> "online"
+        NetworkStatus.OFFLINE -> "offline"
+        NetworkStatus.UNKNOWN -> "unknown"
+    }
+}
+
+fun mapFromOfflineStatus(offlineStatus: OfflineStatus): ReadableMap =
+    nativeMapOf {
+        putString("networkStatus", mapFromNetworkStatus(offlineStatus.sdk.networkStatus))
```

Comment:
I think we'll want to include `offlinePaymentsCount` and `offlinePaymentAmountsByCurrency` here.